tests/integration/security/util/reachability/context.go

```diff
@@ -172,11 +179,17 @@ func Run(testCases []TestCase, ctx framework.TestContext, apps *util.EchoDeploym
 								if c.Include(src, opts) {
 									expectSuccess := c.ExpectSuccess(src, opts)
 
-									subTestName := fmt.Sprintf("with scheme %s to %s:%s%s",
+									// TODO failure from naked -> VM is 1s due to the fact service is empty
```

Comment:
is this TODO still relevant? I saw you just skipped the naked -> VM above.

Reply:
Probably not since we are skipping it now

Reply:
can you remove it? approved with /hold, feel free to merge after removing.